<launch>

  <arg name="scan_topic" default="scan" />

  <node pkg="hector_mapping" type="hector_mapping" name="hector_mapping" output="screen">
    <param name="pub_map_odom_transform" value="true"/>
    <param name="map_frame" value="map" />
    <param name="base_frame" value="odom" />
    <param name="odom_frame" value="odom" />
    <param name="map_resolution" value="0.05" />
    <param name="map_size" value="1024" />
    <param name="laser_max_dist" value="3.9" />
    <param name="tf_map_scanmatch_transform_frame_name" value="odom_laser" />
    <remap from="scan" to="$(arg scan_topic)"/>
  </node>
</launch>
